_includes/javascript-docs-tracking.html

```diff
@@ -0,0 +1,28 @@
+<script>
+  (function() {
+    function sendToGa() {
+      window.dataLayer = window.dataLayer || []; // Provide empty array to push into to avoid errors in development
+      window.dataLayer.push({
+        event: 'page-view',
+        eventLabel: 'page-view',
+      });
+    }
+
+    function sendToBE() {
+      fetch('https://zapier.com/api/v3/events', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
+          message: 'navigate'
+        }),
+      });
+    }
+
+    document.addEventListener('DOMContentLoaded', function() {
+      sendToGa();
```

Comment:
Sending tracking information once the page is loaded.

Reply:
This is the `prod` GTM code. How can we test this?